### PR TITLE
Add Orbital Sync template

### PR DESCRIPTION
**Pull request by mattwebbio**
### Before submitting a *pull request* make sure you have

- [x] Checked out our [guide](https://selfhosters.net/docker/templating/templating/) on writing a template. Especially the part about [removing redudnant information](https://selfhosters.net/docker/templating/templating/#114-shave-off-the-xml).

---

### Description of your *pull request* and other information

This pull request adds a new template for [Orbital Sync](https://github.com/mattwebbio/orbital-sync/), a small utility that synchronizes Pi-hole DNS server configurations for high availability and redundancy. A few members of the community requested this template, and one of them [tested it as working](https://github.com/mattwebbio/orbital-sync/pull/51#issuecomment-1921513971).

Let me know if this needs any changes or if you have any questions. Thanks 🙂

### CODE REVIEW

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:38]:
```suggestion
    <Config Name="interval minutes" Target="INTERVAL_MINUTES" Default="30" Mode="" Description="How long to wait between synchronizations. Defaults to five minutes. Remember that the DNS server on your secondary servers restarts everytime a sync is performed. i.e. '5', '30', or '1440'" Type="Variable" Display="always" Required="false" Mask="false">30</Config>
```

Any field with a `Default` should have that default auto-populated.

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:39]:
```suggestion
    <Config Name="sync whitelist" Target="SYNC_WHITELIST" Default="true" Mode="" Description="Copies the whitelist" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:40]:
```suggestion
    <Config Name="sync regex whitelist" Target="SYNC_REGEX_WHITELIST" Default="true" Mode="" Description="Copies the regex whitelist" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:41]:
```suggestion
    <Config Name="sync blacklist" Target="SYNC_BLACKLIST" Default="true" Mode="" Description="Copies the blacklist" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:42]:
```suggestion
    <Config Name="sync regexlist" Target="SYNC_REGEXLIST" Default="true" Mode="" Description="Copies the regex blacklist" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:43]:
```suggestion
    <Config Name="sync adlist" Target="SYNC_ADLIST" Default="true" Mode="" Description="Copies the adlist" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:44]:
```suggestion
    <Config Name="sync client" Target="SYNC_CLIENT" Default="true" Mode="" Description="Copies clients" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:45]:
```suggestion
    <Config Name="sync group" Target="SYNC_GROUP" Default="true" Mode="" Description="Copies groups" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:46]:
```suggestion
    <Config Name="sync audit log" Target="SYNC_AUDITLOG" Default="false" Mode="" Description="Copies the audit log" Type="Variable" Display="advanced" Required="false" Mask="false">false</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:47]:
```suggestion
    <Config Name="sync static dhcp leases" Target="SYNC_STATICDHCPLEASES" Default="false" Mode="" Description="Copies static dhcp leases" Type="Variable" Display="advanced" Required="false" Mask="false">false</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:48]:
```suggestion
    <Config Name="sync local dns records" Target="SYNC_LOCALDNSRECORDS" Default="true" Mode="" Description="Copies local DNS records" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:49]:
```suggestion
    <Config Name="sync local cname records" Target="SYNC_LOCALCNAMERECORDS" Default="true" Mode="" Description="Copies local CNAME records" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:50]:
```suggestion
    <Config Name="sync flush tables" Target="SYNC_FLUSHTABLES" Default="true" Mode="" Description="Clears existing data on the secondary (copy target) Pi-hole" Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:51]:
```suggestion
    <Config Name="notify on success" Target="NOTIFY_ON_SUCCESS" Default="false" Mode="" Description="Send a notification if a sync completes successfully." Type="Variable" Display="advanced" Required="false" Mask="false">false</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:52]:
```suggestion
    <Config Name="notify on failure" Target="NOTIFY_ON_FAILURE" Default="true" Mode="" Description="Send notifications if a sync fails for any reason." Type="Variable" Display="advanced" Required="false" Mask="false">true</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:53]:
```suggestion
    <Config Name="notify via SMTP" Target="NOTIFY_VIA_SMTP" Default="false" Mode="" Description="Send notifications via email." Type="Variable" Display="advanced" Required="false" Mask="false">false</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:61]:
```suggestion
    <Config Name="verbose mode" Target="VERBOSE" Default="false" Mode="" Description="Increases the verbosity of log output. Useful for debugging." Type="Variable" Display="advanced" Required="false" Mask="false">false</Config>
```

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:62]:
```suggestion
```

TZ is auto-populated, no need to explicitly declare

File: templates/orbital-sync.xml
Comment (nwithan8) [inline templates/orbital-sync.xml:11]: Remove some of these empty elements (`WebUI`, `TemplateURL`, etc.)

**Review by nwithan8 — CHANGES_REQUESTED** [review]
Looks pretty good! Thanks for putting a lot of the "extra" settings under Advanced.

Just some minor recommendations, basically anytime a field has a `Default` value, you should set that value as the visible default in the text entry field so users know what it is.

Also, personal nitpick, could we capitalize/sentence-case the `Name` of all the fields, just to pretty it up?

After that, we'll get this merged in. Great tool!

File: templates/orbital-sync.xml
Comment (mattwebbio) [reply templates/orbital-sync.xml:38]: Is this _in addition_ to `Default="30"` as in your suggestion or is it supposed to in place of:
```suggestion
    <Config Name="interval minutes" Target="INTERVAL_MINUTES" Mode="" Description="How long to wait between synchronizations. Defaults to five minutes. Remember that the DNS server on your secondary servers restarts everytime a sync is performed. i.e. '5', '30', or '1440'" Type="Variable" Display="always" Required="false" Mask="false">30</Config>
```

**Comment by mattwebbio** [comment]
Pushed up all requested changes - let me know if you need anything else :)

File: templates/orbital-sync.xml
Comment (mattwebbio) [reply templates/orbital-sync.xml]: I pushed up the former, but I'm happy to do the latter if necessary!

Comment (nwithan8) [reply templates/orbital-sync.xml]: Yes, in addition.

**Review by nwithan8 — APPROVED** [review]

**Comment by mattwebbio** [comment]
Hi @nwithan8! So sorry for the ping. Do you know whose approval I need to get for this to merge?

File: templates/orbital-sync.xml
Comment (Roxedus) [inline templates/orbital-sync.xml:34]: These are redundant, having some text in the overview on how to add more than 2 would be prefered.

File: templates/orbital-sync.xml
Comment (Roxedus) [inline templates/orbital-sync.xml:50]: These bools can be presented as a drop-down, as done in [vaultwarden](https://github.com/selfhosters/unRAID-CA-templates/blob/98fb52fa952be8b93da94f1e2ac4658a661090cd/templates/vaultwarden.xml#L47) for the Default value. the true default is the first value in the pipe(|) separated list.

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Comment by nwithan8** [comment]
@mattwebbio do you have a chance to make the requested changes?

**Comment by mattwebbio** [comment]
@nwithan8 so sorry, I forgot about this entirely. I do not have time right now - my stuff is packed and I am in the process of moving.

**Comment by nwithan8** [comment]
Seems Orbital Sync was added by a separate maintainer a few months ago, closing this.

**Pull request closed without merging**
